src/app/booking/OOSPlannerSection.js

```diff
@@ -78,19 +88,34 @@ function useAwaitingBookingDeletion({ bookingIdInitial }) {
   return { setRequestNum }
 }
 
-function FilledBooking({ booking }) {
+function FilledBooking({ booking, weekDates }) {
   const { setRequestNum: setAwaitingBookingDeletion } = useAwaitingBookingDeletion({bookingIdInitial: booking.id });
   const { enqueueSnackbar } = useSnackbar();
+  const onSuccess = () => {
```

Comment:
Include in mutation definition. Otherwise I do not unsderstand what is success here, if this function considered separately

Reply:
onSuccess is used in two places

Reply:
onMutationSuccess